client/src/app/tabs/form/FormPreviewToggle.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React from 'react';
+
+import classnames from 'classnames';
+
+import { Fill } from '../../slot-fill';
+
+import DesignIcon from '../../../../resources/icons/Design.svg';
+import ValidateIcon from '../../../../resources/icons/Validate.svg';
+
+import css from './FormPreviewToggle.less';
+
+
+export function FormPreviewToggle(props) {
+
+  const {
+    onCollapsePreview,
+    onOpenPreview,
+    previewOpen
+  } = props;
+
+  return <Fill slot="status-bar__app" group="1_form-preview">
+    <div className={ classnames(css.FormPreviewToggle) }>
+      <button
+        className={ classnames('btn', { 'btn--active': !previewOpen }) }
+        title="Open design"
```

Comment:
I assume "Open design mode" is better when it comes to a11y (respectively "Open validation mode")

Reply:
Updated via b5225cc56ca87456047c39d9986abc83f5931ea1 👍